README.md

```diff
@@ -7,3 +7,136 @@
 [![codecov](https://codecov.io/gh/spacetelescope/stcal/branch/main/graph/badge.svg?token=C1LO00W9CZ)](https://codecov.io/gh/spacetelescope/stcal)
 
 STScI Calibration algorithms and tools.
+
+![STScI Logo](docs/_static/stsci_logo.png)
+
+**STCAL requires Python 3.9 or above and a C compiler for dependencies.**
+
+**Linux and MacOS platforms are tested and supported.  Windows is not currently supported.**
+
+**If installing on MacOS Mojave 10.14, you must install 
+  into an environment with python 3.9. Installation will fail on python 3.10 due
+  to lack of a stable build for dependency ``opencv-python``.**
+
+
+`STCAL` is intended to be used as a support package for calibration pipeline
```

Comment:
I would add a sentence here that mentions Roman and the motivation for this separate package (code common to both JWST + Roman, but could be general use as well), just to make it clearer to someone trying to understand

Reply:
I agree with @cshanahan1

Reply:
I mentioned Roman to clarify STCAL's usage in multiple pipelines.